ui/kotlinx-coroutines-android/build.gradle.kts

```diff
@@ -28,24 +28,24 @@ dependencies {
 
 
     testImplementation("org.smali:baksmali:${version("baksmali")}")
-    "r8"("com.android.tools.build:builder:7.1.0-alpha01")
+    "r8"("com.android.tools.build:builder:8.1.0")
```

Comment:
FTR this change reduced the dex size from `79824` to `75120`.

It will affect our [stats](https://teamcity.jetbrains.com/project/KotlinTools_KotlinxCoroutines?mode=builds&projectTab=stats), it is expected